### ISSUE TITLE
Can not MINT new NFTs

### ISSUE THREAD

**Issue by wzr1337** [joleitner/cardano-car-documents#3]
I get the following error in the terminal:

`cardano-cli: Network.Socket.connect: <socket: 11>: permission denied (Permission denied)error - unhandledRejection: RangeError [ERR_HTTP_INVALID_STATUS_CODE]: Invalid status code: 1`

The UI just refreshes without any warning

**Comment by wzr1337** [comment]
tried to create new policy, but that did not fix it

**Comment by joleitner** [comment]
As described under **4. Adjust node.socket** in the readme, the permissions must be adjusted.

**Comment by wzr1337** [comment]
can not confirm this fix:

```sh
$ sh ./bin/chown_node_socket.sh
OCI runtime exec failed: exec failed: unable to start container process: open /dev/pts/0: operation not permitted: unknown
```

**Comment by wzr1337** [comment]
I do NOT run docker as root, maybe this causes the issue?

https://docs.docker.com/engine/install/linux-postinstall/

https://docs.docker.com/engine/security/rootless/

**Comment by wzr1337** [comment]
specifying user also does not help:

```sh
$ docker-compose exec --user docker app chown node /ipc/node.socket
unable to find user docker: no matching entries in passwd file
```

Is there any doc around the command you use?

**Comment by wzr1337** [comment]
I found out that if I open a shell into the container and change it manually, it works.. we need to automate this process somehow

**Comment by wzr1337** [comment]
@joleitner Please reopen

**Comment by joleitner** [comment]
That's strange that the script doesn't work for you.... Are you running the script normally at the top level of the project? I have also already tried an automation. The problem is that the node.socket is recreated (as root) every time the cardano-node is started. I have not found a solution on how to check that the cardano-node is ready and therefore the node.socket has been created.
Maybe you've got a better idea...

**Comment by wzr1337** [comment]
It so seems as if the socket creation takes a while after booting the container... So running a command on boot would not work except using a loop that checks for 2ipc/node.socket

**Comment by joleitner** [comment]
Exactly, and I didn't want to run a script with a loop... Then I'd rather check myself whether the cardano-node is running successfully.
